.github/workflows/soak-test-start.yml

```diff
@@ -198,6 +198,12 @@ jobs:
             done
           fi
 
+      - name: Wait 10 Minutes for adapters to deply
```

Comment:
@mxiao-cll can we get rid of this since it didn't fix anything?

Reply:
Yes can remove